src/main/java/com/metamx/collections/bitmap/WrappedRoaringBitmap.java

```diff
@@ -126,6 +131,7 @@ public int size()
   @Override
   public void serialize(ByteBuffer buffer)
   {
+    if ( COMPRESS_RUN_ONSERIALIZATION ) bitmap.runOptimize();
```

Comment:
minor formatting, we like to have braces around all if statements to avoid confusion